src/os_auth/local-server.c

```diff
@@ -411,7 +411,7 @@ cJSON* local_remove(const char *id, int purge) {
     w_mutex_lock(&mutex_keys);
 
     if (index = OS_IsAllowedID(&keys, id), index < 0) {
-        merror("ERROR %d: %s.", ERRORS[ENOAGENT].code, ERRORS[ENOAGENT].message);
+        mdebug1("Error code %d: %s.", ERRORS[ENOAGENT].code, ERRORS[ENOAGENT].message);
```

Comment:
I would remove the word `code`. Usually, this kind of logs just say `Error X: message`

Reply:
Done!